staging/src/k8s.io/code-generator/cmd/validation-gen/targets.go

```diff
@@ -286,6 +300,14 @@ func GetTargets(context *generator.Context, args *Args) []generator.Target {
 				},
 			})
 	}
+
+	if args.Lint && len(lintErrs) > 0 {
```

Comment:
I think we should ALWAYS print the lint messages (we always run the lint).  When `--lint` is set, it means "only lint", but lint errors should always be fatal, don't you think?

Reply:
SG. fatal when `--lint` added and warning otherwise.